src/basler_camera_node.cpp

```diff
@@ -82,7 +102,13 @@ void handle_basler_enum_parameter(CInstantCamera& camera, string name, string va
 void handle_basler_parameter(CInstantCamera& camera, XmlRpc::XmlRpcValue& param)
 {
   string type = param["type"];
-  if ("int" == type)
+  if ("boolean" == type)
+  {
+    ROS_ASSERT_MSG(param["value"].getType() == XmlRpcValue::TypeBoolean,
```

Comment:
should be:
    ROS_ASSERT_MSG(param["value"].getType() == XmlRpc::XmlRpcValue::TypeBoolean,